lib/jekyll/converters/smartypants.rb

```diff
@@ -3,9 +3,15 @@
 class Kramdown::Parser::SmartyPants < Kramdown::Parser::Kramdown
   def initialize(source, options)
     super
-    @block_parsers = [:block_html]
+    @block_parsers = [:block_html, :content]
     @span_parsers =  [:smart_quotes, :html_entity, :typographic_syms, :span_html]
   end
+
+  CONTENT_MATCH = %r!^.*\n!
+  def parse_content
+    add_text(@src.scan(CONTENT_MATCH), @tree)
+  end
+  define_parser(:content, %r!^!)
```

Comment:
the regex looks weird to me.. *"match something that starts with nothing"*... 
Also, consistency-wise.. we use regexes with `\A` instead of `^` (not that it matters..)

Reply:
Yeah, it's weird looking, but do you have a better suggestion?

I will use `\A` instead of `^`. Thanks for catching that :+1:

Reply:
I'm searching for the source to `:define_parser`.. can't seem to find it..

Reply:
found it.. placing it here for quick reference..

```ruby
      # Add a parser method
      #
      # * with the given +name+,
      # * using +start_re+ as start regexp
      # * and, for span parsers, +span_start+ as a String that can be used in a regexp and
      #   which identifies the starting character(s)
      #
      # to the registry. The method name is automatically derived from the +name+ or can explicitly
      # be set by using the +meth_name+ parameter.
      def self.define_parser(name, start_re, span_start = nil, meth_name = "parse_#{name}")
        raise "A parser with the name #{name} already exists!" if @@parsers.has_key?(name)
        @@parsers[name] = Data.new(name, start_re, span_start, meth_name)
      end
```

Reply:
We need something that always matches, but never consumes any of the actual string :laughing:

I think `%r!\A!` communicates this as well as anything could, it's just odd.